src/mail-app/contacts/view/ContactView.ts

```diff
@@ -347,9 +347,10 @@ export class ContactView extends BaseTopLevelView implements TopLevelView<Contac
 		)
 	}
 
-	private getHeaderLabel(): TranslationKey {
+	private getHeaderLabel() {
```

Comment:
please add a return type to it (I think it should either be `MaybeTranslation` or `Translation`)